apps/backoffice-v2/src/pages/Entities/components/CaseCreation/hooks/useCaseCreationWorkflowDefinition/selectors/select-workflow-definition-id.ts

```diff
@@ -0,0 +1,8 @@
+import { TFilter } from '@/domains/filters/fetchers';
+
+export const selectWorkflowDefinitionId = (filter: TFilter | null) => {
+  if (!filter) return null;
+
+  // TO DO: Find better place to aquire workflowDefinitionId
```

Comment:
I know it sounds silly, but we should remove the space in the "TO DO" so we could always find it by searching "TODO"